src/main/java/com/microsoft/graph/requests/extensions/AppRoleAssignmentCollectionWithReferencesPage.java

```diff
@@ -7,7 +7,7 @@
 import com.microsoft.graph.http.IRequestBuilder;
 import com.microsoft.graph.core.ClientException;
 import com.microsoft.graph.concurrency.ICallback;
-import com.microsoft.graph.models.extensions.ServicePrincipal;
+import com.microsoft.graph.models.extensions.Device;
```

Comment:
These ServicePrincipal to Device and Group change doesn't make sense. Actually, neither one of the models makes sense here.

Reply:
I noticed that my change from a few months ago from star imports to selective imports sometimes over imports.
But that's not a big problem on it's own.